v9/schema/enum.go

```diff
@@ -50,7 +50,7 @@ func (e *EnumDefinition) SymbolName(symbol string) string {
 }
 
 func (e *EnumDefinition) GoType() string {
-	return generator.ToPublicName(e.name.Name)
+	return generator.ToPublicName(e.name.String())
```

Comment:
@actgardner unsure if this was intentionally using `.Name` instead of `.String()` 😅  
None of the existing tests broke when I made this change so I suspect this is not impactful though would appreciate some additional context around this 👍